src/lib.rs

```diff
@@ -719,7 +719,7 @@ impl EGraph {
         Ok(())
     }
 
-    fn eval_expr(
+    pub fn eval_expr(
```

Comment:
I started using this to eval the Python expr (which looks like `(py-object-hash 123123 123123)`) to the value, so then I can retrieve the Python object from it.